Write Your Own Crontab Tool/Sercice/ValidationService.swift

```diff
@@ -123,6 +123,21 @@ private extension ValidationService {
         
         return error
     }
+    
+    func validateHashtagSeparator(_ value: String, field: Field) -> CustomError? {
```

Comment:
This function's logic is very similar to validateRangeSeparator and validateStepRange. You can do better!